opennms-install/src/main/java/org/opennms/install/Installer.java

```diff
@@ -414,6 +416,10 @@ public void loadProperties() throws IOException {
         //        m_installerDb.setCreateSqlLocation(m_etc_dir + File.separator + "create.sql");
     }
 
+    private void loadScvProperties() {
```

Comment:
May be we can do this directly in JdbcDataSource where we use SCV there by not polluting Installer.